flyteadmin/pkg/config/config.go

```diff
@@ -47,10 +47,10 @@ type DataProxyUploadConfig struct {
 }
 
 type GrpcConfig struct {
-	Port                 int  `json:"port" pflag:",On which grpc port to serve admin"`
-	ServerReflection     bool `json:"serverReflection" pflag:",Enable GRPC Server Reflection"`
-	MaxMessageSizeBytes  int  `json:"maxMessageSizeBytes" pflag:",The max size in bytes for incoming gRPC messages"`
-	EnableGrpcHistograms bool `json:"enableGrpcHistograms" pflag:",Enable grpc histograms"`
+	Port                     int  `json:"port" pflag:",On which grpc port to serve admin"`
+	ServerReflection         bool `json:"serverReflection" pflag:",Enable GRPC Server Reflection"`
+	MaxMessageSizeBytes      int  `json:"maxMessageSizeBytes" pflag:",The max size in bytes for incoming gRPC messages"`
+	EnableGrpcLatencyMetrics bool `json:"enable-grpc-latency-metrics" pflag:",Enable grpc latency metrics. Note Histograms metrics can be expensive on Prometheus servers."`
```

Comment:
this matches the [flytepropeller flag](https://github.com/flyteorg/flyte/blob/5017e6c34e3495e1b455700e3df3c41077d77858/flytepropeller/pkg/controller/config/config.go#L144) at the expense of inconsistent casing - not sure what's preferable here for consistency

Reply:
My vote is consistent casing within admin

Reply:
Personally, I'm not a fan of came case flags, but I agree that I'd rather keep it consistent.

Reply:
thanks folks, updated